tests/lsstseries_tests/test_analysis.py

```diff
@@ -135,6 +135,19 @@ def test_dt_bins():
     assert len(bins) == 11
 
 
+def test_dt_bins_raises_exception():
```

Comment:
This test should raise the code coverage to 100% for this file. But it otherwise unrelated to the core changes here.